app/views/content/non-uk-teachers/visas-for-non-UK-trainees.md

```diff
@@ -0,0 +1,257 @@
+
+A Youth Mobility Scheme visa lasts up to 2 years. You may be able to switch from a Youth Mobility Scheme visa to another visa route (for example, the skilled worker visa) without having to leave the UK. You’ll need to meet the requirements for the new visa route.
+
+### Other visas 
+Depending on your personal circumstances, you may be able to work or study in the UK on alternative visas – for example:
+
+* a [Family visa](https://www.gov.uk/uk-family-visa)
+* a [UK Ancestry visa](https://www.gov.uk/ancestry-visa) 
+* if you are from Hong Kong, a [British National (Overseas) visa](https://www.gov.uk/british-national-overseas-bno-visa)
+
+### Refugees and asylum seekers 
+
+If you have been granted refugee status, you can apply for teacher training – either salaried or fee-paying – and will usually be eligible for [financial support to train to teach on a fee-paying course](/non-uk-teachers/fees-and-funding-for-non-UK-trainees).
+
+If you are in the process of seeking asylum, you should refer to your immigration bail conditions to see if you have permission to study on a teacher training course. You will not usually be eligible for financial support to train to teach on a fee-paying course. You are also unlikely to be able to undertake salaried teacher training while your asylum claim is considered, because asylum seekers do not usually have the right to work, although [there are some exceptions](https://www.gov.uk/government/publications/working-whilst-an-asylum-claim-is-considered/working-in-the-uk-whilst-an-asylum-case-is-considered).
+
+Learn more about:
+
+* [claiming asylum in the UK](https://www.gov.uk/claim-asylum) if you are an asylum seeker
+* [support for British and non-British nationals in Afghanistan](https://www.gov.uk/guidance/support-for-british-and-non-british-nationals-in-afghanistan)
+* [support for Ukrainian nationals](/non-uk-teachers/ukraine)
+
+## Visa fees and other charges
+
+You will usually need to show proof of a certain level of savings or income, depending on which visa you are applying for. You’ll be guided through this process when you apply for your visa.
+
+### Immigration Health Surcharge (IHS)
+
+You may need to [pay for UK healthcare](https://www.gov.uk/healthcare-immigration-application) as part of your immigration application – this is called the IHS.
+
+However, students can usually claim a refund of the IHS if they:
+
+* are a full-time student in UK higher education
+* have a visa starting on or after 1 January 2021
+* have a European Health Insurance Card (EHIC) issued in an EU country
+* do not work in the UK
+
+Learn more about [getting an IHS refund](https://www.gov.uk/guidance/immigration-health-surcharge-for-eu-students-in-the-uk).
+
+### Visa costs
```

Comment:
fees or costs? I think we need to be consistent.

Reply:
yep good spot